Cookbook/Technical-Documents/XcodeTips.md

```diff
@@ -27,4 +27,10 @@ When merging develop into your branch, if the project file has been changed in b
 ```
 - Then whenever you run into a project file conflict you can resolve it with:
 `git mergetool --tool=mergepbx [Project File]`
-ex: `git mergetool --tool=mergepbx Babylon.xcodeproj/project.pbxproj`
\ No newline at end of file
+ex: `git mergetool --tool=mergepbx Babylon.xcodeproj/project.pbxproj`
+
+
+### Console logs
+
+- to disable autolayout warnings add `-_UIConstraintBasedLayoutLogUnsatisfiable NO` to the scheme launch arguments
```

Comment:
Why would we want to disable them? To avoid polluting the console? If so, do you think it would make sense to specify this just below `Console Logs`?

Reply:
Mainly because the majority of the time we're not interacting with UIKit directly - they're not a result of the development that we are currently doing. 
We should definitely keep them enabling if w e're developing a component though.